packages/php-wasm/universal/src/lib/php-browser.ts

```diff
@@ -111,7 +111,8 @@ export class PHPBrowser implements RequestHandler {
 	get documentRoot() {
 		return this.requestHandler.documentRoot;
 	}
-	#setCookies(cookies: string[]) {
```

Comment:
I don't have a strong opinion here, but I'm curious what problem does this solve?

Reply:
That allows us to share cookies between multiple PHP instances. Without that code, we'll get some 403s during handoff between instances due to mismatched authentication tokens.

https://github.com/WordPress/playground-tools/pull/110/files#diff-eeed5d391a89564a5ff176b28d0ab5a31e01b94fb75f4ccc1d67f80297586595R194-R199

Reply:
Ah I gotcha. And because this notion of a `private` method is scoped to an instance and not to a class, the method needs to be public. Thank you for explaining!